bin/start

```diff
@@ -37,11 +39,13 @@ RUN_IN_DOCKER=false
 DEMO=false
 CONJUR_DEPLOYMENT=oss
 SUMMON_ENV=gke
+DEV_ENV=false
 
 while true ; do
   case "$1" in
     --docker ) RUN_IN_DOCKER=true ; shift ;;
     --demo ) DEMO=true ; shift ;;
+    --dev ) DEV_ENV=true ; shift ;;
```

Comment:
will `--docker` work with `--dev`? If not, then let's restrict it to verify they are not both true